releng/org.eclipse.xtext.contributor/Xtext.setup

```diff
@@ -50,6 +50,10 @@
       value="http://services.typefox.io/open-source/jenkins/job/xtext-eclipse/job/master/lastStableBuild/artifact/build/p2-repository/"/>
   <setupTask
       xsi:type="setup:VariableTask"
+      name="xtend.nightly.composite"
+      value="http://services.typefox.io/open-source/jenkins///job/xtext-xtend/job/master/lastSuccessfulBuild/artifact/build/p2-repository/"/>
```

Comment:
I would avoid the triple ///
This is probably just cosmetic, but with p2 you never know ;)